### ISSUE TITLE
Can we use  timeseries of Pointclouds to classify 

### ISSUE THREAD

**Issue by sunilsarolkarcds** [facebookresearch/pytorch3d#1783]
## How can we use Pytorch3d to detect movement of pointcloud in space

We are working on a project to implement sign language translation using neural network, as a part of this we have used existing openpose models to convert body and hand movement in xy points and edges. So we have kind of timeseries of  thes xy points and edges that represent specific word or phrase. We were thinking of initialize Pointcloud using these x,y coordinates and use them for further processing using pytorch.

Now we are looking to use Pytorch that would allow us to do the classification of these timeseries of pointclouds to label.  We have checked your implementation of GenericModel , how can we use the GenericModel to feed to timeseries implementation 

Can someone help us understand if we can use Pytorch3d to  generate the neural network for this specific purpose.

**Comment by bottler** [comment]
I'm sorry we can't help with modelling questions.
